test/test_ops.py

```diff
@@ -860,20 +865,11 @@ def test_bbox_same(self):
         exp_xyxy = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
                                 [10, 15, 30, 35], [23, 35, 93, 95]], dtype=torch.float)
 
-        box_same = ops.box_convert(box_tensor, in_fmt="xyxy", out_fmt="xyxy")
-        self.assertEqual(exp_xyxy.size(), torch.Size([4, 4]))
```

Comment:
I think that we shoudn't remove `self.assertEqual(exp_xyxy.size(), torch.Size([4, 4]))` as it's not the same as comparing `exp_xyxy.size()` vs `box_tensor.size()`, so I'll put these back